snippets/dockerfile.json

```diff
@@ -110,5 +110,14 @@
             "$0"
         ],
         "description": "ONBUILD"
+    },
+    "HEALTHCHECK": {
+        "prefix": "HEALTHCHECK",
+        "body": [
+            "HEALTHCHECK --interval=30s --timeout=30s --retries=3 CMD ${command}"
```

Comment:
Those actually are the default values for those options. I wasn't sure of the best way to handle including the options for reference so I went with that. However, I'm open to suggestions. 

None of the other entities had similar situations for me to look at as a reference so I wasn't sure if `--interval=${interval} --timeout=${timeout} --retries=${retries}` would work, but I also didn't want to leave them out completely because then a user would have to go look them up if they needed them which seems to defeat the purpose of a helper.